src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -23,11 +25,47 @@ export interface IMutationMapperProps {
     pmidCache?:PmidCache;
 }
 
+const MISSENSE_COLOR = "#008000";
+const TRUNCATING_COLOR = "#000000";
+const INFRAME_COLOR = "#8B4513";
+const OTHER_COLOR = "#8B00C9";
+
 @observer
 export default class MutationMapper extends React.Component<IMutationMapperProps, {}>
 {
+    private handlers:any;
+    private mutationTable:ResultsViewMutationTable;
+    private lollipopPlot:LollipopMutationPlot;
+
     constructor(props: IMutationMapperProps) {
         super(props);
+
+        this.handlers = {
+            resetDataStore:()=>{
+                this.props.store.dataStore.reset();
+                this.mutationTable && this.mutationTable.resetFilterText();
+                this.lollipopPlot && this.lollipopPlot.resetSelectedPositions();
```

Comment:
Ideally, resetting the data store would automatically reset selected positions within the lollipop plot, since it already observes the changes in data store.